backend/app/Services/Traceability/TraceabilityService.php

```
@@ -57,6 +63,225 @@ public function forwardTrace(MaterialLot $lot): array
+
+        for ($depth = 0; ! empty($frontier); $depth++) {
+            if ($depth >= self::MAX_DEPTH) {
+                $truncated = true;
+                break;
+            }
+
+            foreach ($frontier as $id) {
+                $visitedLotIds[$id] = true;
+            }
+
+            $consumptions = BatchStepLotConsumption::query()
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\BatchStepLotConsumption::query().


(staticMethod.notFound)